### ISSUE TITLE
Social media sharing buttons?

### ISSUE THREAD

**Issue by leason** [josephhutch/aether#22]
If I wanted to enable twitter/linkedin sharing for posts it appears I'd need to update the theme.  I'm happy to make this enhancement but want to make sure this isn't already in the works.  Is anyone else working on social media sharing yet?

**Comment by josephhutch** [comment]
You are correct that this is not implemented and I don't think anyone is working on it.  If you adding social media sharing I would really appreciate it 👍

**Comment by leason** [comment]
No problem - I've got a twitter share button working already.  I need to style it up and add LinkedIn and then I'll submit a PR.

**Comment by leason** [comment]
What's the best way to generate the minified CSS, @josephhutch ?  I can write CSS but I've been out of that game a while.  For dev I updated head.html to refer to styles.css but before I submit the PR I want to fix that.

**Comment by josephhutch** [comment]
I use the ([uglifycss command line tool](https://www.npmjs.com/package/uglifycss) to minify the css files into one sheet.  Thank you for you work on this!
```
 $ uglifycss style.css xcode.css latolatinfonts.css --output main.min.css
```
